project-management/agile-opensource-workflow.md

```diff
@@ -29,30 +33,34 @@ The tasks and user stories are prioritised (with the input of members of the aut
 
 The issues that correspond to the backlog are transferred to the Fujitsu team's fork of the GitHub repository.
 
+### 4. Completion of tasks
+
 Completed tasks are submitted as pull requests to the main repository.
 
 The remaining user stories remain open as issues on GitHub for anyone to develop or contribute to on an ad hoc continual basis.
```

Comment:
continuos

Reply:
```suggestion
The remaining user stories remain open as issues on GitHub for anyone to develop or contribute to on an ad hoc continuous basis.
```

Reply:
I'll have to commit this separately as I don't have permission to commit to @EllenDevereux's fork.

Reply:
Done